src/identify/index.js

```diff
@@ -76,7 +79,7 @@ class IdentifyService {
 
         await pipe(
           [{
-            listenAddrs: this.peerInfo.multiaddrs.toArray().map((ma) => ma.buffer),
+            listenAddrs: this.addresses.listen.map((ma) => ma.buffer),
```

Comment:
We should make a note to update this when the Address Manager is in place. We really shouldn't be using `addresses.listen` here, but should instead be using `addresses.announce`

Reply:
yep, already being done on #612 :)